test/website-docs.spec.js

```diff
@@ -51,4 +51,18 @@ describe('docs', () => {
   it('has headers for every required sub-option', () => {
     expect(headers3).toEqual(expectedOptions3);
   });
+
+  // Checking relatedOptions field in definitions
+  let relatedOptions = options
+    .filter(option => option.relatedOptions)
+    .map(option => option.relatedOptions)
+    .sort();
+
+  relatedOptions = [].concat(...relatedOptions);
```

Comment:
What does this achieve?

Reply:
The last line converts the 2d matrix to 1d to have a valid subset for search. E.g. `[['enabled', 'packageRules']]` => `['eanbled', 'packageRules']`